me/index.php

```diff
@@ -1,24 +1,278 @@
+                </div>
+                <div class="col-md-4">
+                    <h3 class = "profileText"><?php /*echo $userinfo['discord']; */?></h3>
+                </div>
+                <div class="col-md-4"><h3 class = "button">Edit</h3></div>
+            </div>
+            <div class="row">
+                <div class="col-md-4">
+                    <h4 class = "profileText" style = "text-align:right">LinkedIn Account:</h4>
+                </div>
+                <div class="col-md-4">
+                    <h3 class = "profileText"><?php /*echo $userinfo['linkedin']; */?></h3>
+                </div>
+                <div class="col-md-4"><h3 class = "button">Edit</h3></div>-->
+            </div>
+            <div class="row">
+                <div class="col-md-12 orangeBox">
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h4 class = "profileText" style = "text-align:right">UNT EUID:</h4>
+                        </div>
+                        <div class="col-md-4">
+                            <h4 class = "profileText" style = "text-align:center"><?php echo $userinfo['UNTEUID']; ?></h4>
+                        </div>
+                        <div class="col-md-4">
+<!--                            <h3 class = "button">Edit</h3>-->
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h4 class = "profileText" style = "text-align:right">Graduation Date:</h4>
+                        </div>
+                        <div class="col-md-4">
+                            <h4 class = "profileText" style = "text-align:center"><?php echo $userinfo['GradDate']; ?></h4> <!--Check variable name(s) and change accordingly-->
+                        </div>
+                        <div class="col-md-4">
+<!--                            <h3 class = "button">Edit</h3>-->
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-12"><br></div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-3" style = "left:14%">
+                            <h4 class = "profileText">CampusLabs Member</h4>
+                        </div>
+                        <div class="col-md-3" style = "left:13%">
+                            <input type="checkbox"  name="campusLabsBox" value=<?php echo $userinfo['IsCampusLabMember']; ?>>
+                        </div>
+                        <div class="col-md-3">
+                            <h4 class = "profileText" style = "text-align:right">Good Standing</h4>
+                        </div>
+                        <div class="col-md-3">
+                            <input type="checkbox"  name="goodStandingBox" value=<?php echo $userinfo['IsGoodStanding']; ?>>

```

Comment:
The proper indices to use from `$userinfo` are `unteuid` and `grad_year` + `grad_term` (I'm assuming `grad_term` uses the constants defined in the `Semester` class in `untrobotics.php` ), For Good Standing, use `$untrobotics->is_user_in_good_standing($userinfo)`. I don't see anything that stores Campus Labs/OrgSync info though.